### PR TITLE
Add missing entries in CHANGELOG.adoc

### PR DESCRIPTION
**Pull request by B3nedikt**

### CODE REVIEW

File: CHANGELOG.adoc [hunk 1]
```diff
@@ -10,11 +10,23 @@ and this project adheres to https://semver.org/spec/v2.0.0.html[Semantic Version
 
 == Unreleased
 
+## [1.0.0] - 2022-10-07
+### Added
```
Comment (Mogztter) [inline CHANGELOG.adoc:14]:
```suggestion
```

File: CHANGELOG.adoc [hunk 2]
```diff
@@ -10,11 +10,23 @@ and this project adheres to https://semver.org/spec/v2.0.0.html[Semantic Version
...
 === icon:heartbeat[] Changed
 
-* Use 0.11.x as default version for Chartist.js instead of latest (#28)
\ No newline at end of file
+* Use 0.11.x as default version for Chartist.js instead of latest (#28)
+* Update c3.js defaults to latest version 0.7.20 (#32) 
+
+=== icon:wrench[] Fixed
+
+* Fixed ")" in top left corner
```
Comment (Mogztter) [inline CHANGELOG.adoc:30]: We can remove this fix since it wasn't part of any release

File: CHANGELOG.adoc [hunk 3]
```diff
@@ -10,11 +10,23 @@ and this project adheres to https://semver.org/spec/v2.0.0.html[Semantic Version
...
 === icon:rocket[] Added
 
 * Introduce a changelog (#30)
 * Added support for using the JavaScript libraries offline (#25)
+* Add captioned title on charts (#18)
+* Improved documentation
+* Added attributes to define chart resources directory (#26)
+* Introduce visual regression tests (#34)
 
 === icon:heartbeat[] Changed
```
Comment (Mogztter) [inline CHANGELOG.adoc:23]: we should include https://github.com/asciidoctor/asciidoctor-chart/pull/17

File: CHANGELOG.adoc [hunk 4]
```diff
@@ -10,11 +10,23 @@ and this project adheres to https://semver.org/spec/v2.0.0.html[Semantic Version
...
 
-* Use 0.11.x as default version for Chartist.js instead of latest (#28)
\ No newline at end of file
+* Use 0.11.x as default version for Chartist.js instead of latest (#28)
+* Update c3.js defaults to latest version 0.7.20 (#32) 
+
+=== icon:wrench[] Fixed
```
Comment (Mogztter) [inline CHANGELOG.adoc:28]: We can add this bug fix: https://github.com/asciidoctor/asciidoctor-chart/pull/13

Comment (Mogztter) [inline CHANGELOG.adoc:28]: We should include https://github.com/asciidoctor/asciidoctor-chart/pull/24
It includes assets only when a chart engine is used (previously all assets were included for all engines)

Comment (B3nedikt) [reply CHANGELOG.adoc:32]: Done

Comment (B3nedikt) [reply CHANGELOG.adoc:32]: Done.

File: CHANGELOG.adoc (hunk 3 above)
Comment (B3nedikt) [reply CHANGELOG.adoc:23]: Split it between Changed and Fixed cause it was both bugfixing and general changes

File: CHANGELOG.adoc (hunk 2 above)
Comment (B3nedikt) [reply CHANGELOG.adoc]: Removed it.

File: CHANGELOG.adoc [hunk 5]
```diff
@@ -10,11 +10,27 @@ and this project adheres to https://semver.org/spec/v2.0.0.html[Semantic Version
...
 == Unreleased
 
+
 === icon:rocket[] Added
 
 * Introduce a changelog (#30)
 * Added support for using the JavaScript libraries offline (#25)
+* Add captioned title on charts (#18)
+* Improved documentation
+* Added attributes to define chart resources directory (#26)
```
Comment (Mogztter) [inline CHANGELOG.adoc:20]:
```suggestion
* Add attributes to define chart resources directory (#26)
```

File: CHANGELOG.adoc [hunk 6]
```diff
@@ -10,11 +10,27 @@ and this project adheres to https://semver.org/spec/v2.0.0.html[Semantic Version
 
 == Unreleased
 
+
 === icon:rocket[] Added
 
 * Introduce a changelog (#30)
 * Added support for using the JavaScript libraries offline (#25)
```
Comment (Mogztter) [inline CHANGELOG.adoc:17]:
```suggestion
* Add support for using the JavaScript libraries offline (#25)
```

File: CHANGELOG.adoc [hunk 7]
```diff
@@ -10,11 +10,27 @@ and this project adheres to https://semver.org/spec/v2.0.0.html[Semantic Version
...
 === icon:heartbeat[] Changed
 
-* Use 0.11.x as default version for Chartist.js instead of latest (#28)
\ No newline at end of file
+* Use 0.11.x as default version for Chartist.js instead of latest (#28)
+* Update c3.js defaults to latest version 0.7.20 (#32) 
+* Upgrade Chart.js from 1.0.2 to 3.7.0 and update code accordingly (#17)
+* Use https protocol (instead of http) when loading scripts from cdnjs.cloudflare.com (#17)
+* Apply RuboCop to asciidoctor-chart.gemspec (#17)
+* Add rubygems_mfa_required in gemspec (#17)
```
Comment (Mogztter) [inline CHANGELOG.adoc:30]:
```suggestion
* Enable multi-factor authentication when publishing to https://rubygems.org/ (#17)
```

File: CHANGELOG.adoc [hunk 8]
```diff
@@ -10,11 +10,27 @@ and this project adheres to https://semver.org/spec/v2.0.0.html[Semantic Version
...
+
+=== icon:wrench[] Fixed
+
+* Fixed issue (#12) that we can only have one chartjs graph per page with (#13)
```
Comment (Mogztter) [inline CHANGELOG.adoc:34]:
```suggestion
* Fix an issue to allow more than one Chart.js graphs per page (#13)
```

File: CHANGELOG.adoc [hunk 9]
```diff
@@ -10,11 +10,27 @@ and this project adheres to https://semver.org/spec/v2.0.0.html[Semantic Version
...
+* Include assets (styles, scripts)  when an chart engine is used (#24)
+* Fixed issue with chart.js graph overlaping text when width is specified (#8) with (#17)
```
Comment (Mogztter) [inline CHANGELOG.adoc:36]:
```suggestion
* Fix an issue with Chart.js where text was overlapping the graph when width was specified (#17)
```

**Comment by Mogztter** [comment]
Thanks @B3nedikt
I left a few nitpicks but overall that's 👍🏻

**Pull request merged**